test/02_test/test/spam_test.py

```diff
@@ -1,9 +1,33 @@
+from __future__ import print_function
+import os
+import sys
 from unittest import TestCase
 
 import spam
 
 
+def normalize_path(path_str):
+    """because of windows short path"""
+    return os.path.normcase(path_str).replace("vssadm~1", "vssadministrator")
+
```

Comment:
Eesh, is there any way around this? It seems like this could break quite easily.

Maybe instead of the 'path in path' below we could use a function `path_contains(parent, child)`, like:

```python
def path_contains(parent, child): 
    parent = os.path.abspath(parent) 
    child = os.path.abspath(child) 

    while child != os.path.dirname(child): 
        child = os.path.dirname(child)  
        if os.path.samefile(parent, child): 
            return True 
    return False
```

Reply:
I will check this but i use operator `in` because on macos path looks like (`/private` prefix)
```
('=', '/private/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/tmp0gr8ip8m/bin/python')
('=', '/private/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/tmp0gr8ip8m/lib/python2.7/site-packages/spam.so')
('=', '/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/tmp0gr8ip8m')
```